DuckDuckGo.Fluent.Plugin/DuckResult.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+// ReSharper disable NonReadonlyMemberInGetHashCode
+
+namespace DuckDuckGo.Fluent.Plugin
+{
+    public class DuckResult
+    {
+        public string Info { get; set; }
+        public string ResultType { get; set; }
+        public string SourceUrl { get; set; }
+        public string SearchedText { get; set; }
+        public ResultType SearchResultType { get; set; }
+        public bool IsPinned { get; set; }
+
+        public override bool Equals(object obj)
+        {
+            return obj is DuckResult duckResult && (duckResult.Info, duckResult.ResultType, duckResult.SourceUrl,
```

Comment:
FYI - here you create two `Tuple`s of 6 elements for `Equals`. This is considerable less efficient than normally compare all of the properties

Reply:
Now I'm comparing all of the properties except `Score` as I don't think it's necessary to count that in, without using any tuples. Please verify this.